lib/jekyll/admin.rb

```diff
@@ -1,7 +1,26 @@
 require "jekyll/admin/version"
-
+require "jekyll/admin/admin_servlet"
+require "jekyll/admin/api_servlet"
+require "jekyll/admin/api_handler"
+require "jekyll/admin/configuration/handler"
+require "jekyll/admin/configuration/servlet"
+require "jekyll/admin/pages/handler"
+require "jekyll/admin/pages/servlet"
```

Comment:
If there's a lot of sub-files (which is fine), it may make sense to use [autoload](http://www.rubyinside.com/ruby-techniques-revealed-autoload-1652.html), e.g.:

```ruby
 module Jekyll
    module Admin
      autoload :AdminServlet, "jekyll/admin/admin_servlet"
    end
end
```

That way classes are lazy loaded, which may speed up the initial load a bit.